commands/files/serialfile.go

```diff
@@ -69,8 +70,8 @@ func (f *serialFile) NextFile() (File, error) {
 	f.files = f.files[1:]
 
 	// open the next file
-	fileName := fp.Join(f.name, stat.Name())
-	filePath := fp.Join(f.path, stat.Name())
+	fileName := path.Join(f.name, stat.Name())
+	filePath := path.Join(f.path, stat.Name())
```

Comment:
In go-ipfs, if the path string comes cli, yes.

But iirc @whyrusleeping wanted to make `commands/files` into standalone generic file traversing/unmarshal util in #1648, which could be no.